examples/apps/heart_disease_eval/client/heart_gui.py

```diff
@@ -0,0 +1,493 @@
+
+	def get(self):
+		try:
+			return float(self.entry.get())
+		except ValueError:
+			return None
+
+	def enable(self):
+		self.entry.config(state=tk.NORMAL)
+
+	def disable(self):
+		self.entry.config(state=tk.DISABLED)
+
+#Radio button
+class radio:
+	# Options is a list of text-value pairs
+	def __init__(self, master, name, options):
+		global cur_row
+		if not all(len(tup)==2 for tup in options):
+			print("ERROR: Mismatched text-value pairs")
+			exit(1)
+
+		self.var = tk.IntVar()
+		self.var.set(None)
+		label = tk.Label(master, text=name)
+		label.grid(row=cur_row, column=0, pady=(5,0), sticky="e")
+
+		self.button_list = []
+		for i in range(len(options)):
+			button = tk.Radiobutton(master, text=options[i][0], 
+				variable=self.var, value=options[i][1])
+			self.button_list.append(button)
+			if i==0:
+				button.grid(row=cur_row, column=1, pady=(5,0), sticky="w")
+			else:
+				button.grid(row=cur_row, column=1, sticky="w")
+			cur_row += 1
+
+	def get(self):
+		try:
+			return self.var.get()
+		except tk.TclError:
+			return None
+
+	def enable(self):
+		for button in self.button_list:
+			button.config(state=tk.NORMAL)
+
+	def disable(self):
+		for button in self.button_list:
+			button.config(state=tk.DISABLED)
+
+class resultWindow(tk.Toplevel):
+	def __init__(self, parent, message):
+		tk.Toplevel.__init__(self, parent)
+		self.config(background="pale goldenrod")
+		self.parent = parent
+		# Lock main window
+		self.transient(parent)
+		self.grab_set()
+		self.initial_focus = self
+		self.initial_focus.focus_set()
+		self.title("Evaluation Result")
+		self.protocol("WM_DELETE_WINDOW", self.close)
+
+		# Main content
+		self.main_frame = tk.Frame(self, background="pale goldenrod")
+		self.main_frame.pack()
+
+		self.frame1 = tk.Frame(self.main_frame)
+		self.frame1.pack(side=tk.LEFT)
+		self.result_text = tk.StringVar()
+		self.label = tk.Label(self.frame1, textvariable=self.result_text,
+			width=45, background="pale goldenrod")
+		default_font = font.Font(font="TkDefaultFont")
+		new_font = default_font
+		new_font.config(weight=font.BOLD)
+		self.label.config(font=new_font)
+		self.label.pack()
+		self.frame2 = tk.Frame(self.main_frame, background="pale goldenrod")
+		self.frame2.pack(side=tk.LEFT)
+
+		# JSON sidebar
+		self.request_button = tk.Button(
+			self.frame2, text="View Request", command=self.request)
+		self.request_button.pack(fill=tk.X, padx=(0,10), pady=(10,0))
+
+		self.result_button = tk.Button(
+			self.frame2, text="View Result", command=self.result)
+		self.result_button.pack(fill=tk.X, padx=(0,10),pady=(10,0))
+
+		self.receipt_button = tk.Button(
+			self.frame2, text="View Receipt", command=self.receipt)
+		self.receipt_button.pack(fill=tk.X, padx=(0,10),pady=(10,0))
+
+		# Close button
+		self.close_button = tk.Button(self, text="Close", command=self.close)
+		self.close_button.pack(pady=(0,5))
+
+		self.evaluate(message)
+
+	# Submit work order and retrieve result
+	def evaluate(self, message):
```

Comment:
I would suggest to keep GUI specific code and non-GUI code in a separate class and file.